core/color_names.inc

```diff
@@ -0,0 +1,1228 @@
+// Names from https://en.wikipedia.org/wiki/List_of_colors (through https://raw.githubusercontent.com/SuperUserNameMan/color_to_name/616a7cddafefda91478b7bc26167de97fb5badb1/godot_version.gd), slightly edited and normalized
+static Map<String, Color> _named_colors;
+static void _populate_named_colors() {
+	if(!_named_colors.empty()) return;
+	_named_colors.insert("absolutezero", Color(0.00, 0.28, 0.73 ));
```

Comment:
I think almost all of these colors should be thrown away and replaced by HTML ones. I found info that all modern browsers support about 140 common colors. I don't think that "awsome", "absolutezero" (no, that's not 0,0,0) or "maximumgreen" with green set to 55 percent is good choice. So maybe use these 140 standard ones instead of 1200+?